can_tools/scrapers/__init__.py

```diff
@@ -54,10 +54,12 @@
 from can_tools.scrapers.official.federal.HHS.hhs_state import (
     HHSReportedPatientImpactHospitalCapacityState,
 )
-from can_tools.scrapers.official.FL.fl_vaccine import (
-    FloridaCountyVaccine,
-    FloridaCountyVaccineDemographics,
-)
+
+# Scrapers disabled due to source data no longer being updated
+# from can_tools.scrapers.official.FL.fl_vaccine import (
+#     FloridaCountyVaccine,
+#     FloridaCountyVaccineDemographics,
+# )
```

Comment:
Is there a reason to keep these around instead of deleting them?

Reply:
nope! no real reason--I'll delete them